src/reflex.cpp

```diff
@@ -2182,7 +2258,7 @@ void Reflex::write_lexer()
   std::string lex = options["lex"];
```

Comment:
Also add here:
~~~
std::string token_type = options["token_type"].empty() ? "int" : options["token_type"]
~~~
and use `token__type` at line 2424 below.